frontend/public/components/events.jsx

```diff
@@ -136,31 +137,88 @@ export class EventsList extends React.Component {
     super(props);
     this.state = {
       type: 'all',
-      kind: 'all',
       textFilter: '',
+      selected: ['All'],
     };
   }
 
+  updateSelected = (selection) => {
+    let { selected } = this.state;
```

Comment:
Because the selected items are shown in the toolbar area as well and that area allows you to remove the items, I have to keep the selected items at the parent.